app/models/server/models/LivechatAgentActivity.js

```diff
@@ -4,9 +4,8 @@ export class LivechatAgentActivity extends Base {
 	constructor(...args) {
 		super(...args);
 
-		this.tryEnsureIndex({ date: 1 });
```

Comment:
I think we still need this index since I found it getting used [here](https://github.com/RocketChat/Rocket.Chat/blob/ee120705befb1f1be89bed7eec2b106673ffd2ea/app/models/server/raw/LivechatAgentActivity.ts#L78). Let me know if I'm missing something here